subworkflows/local/snv_annotation/tests/main.nf.test.snap

```diff
@@ -17,19 +17,6 @@
             ],
             "## Using API version 110, DB version 110",
             [
-                "## 1000genomes version phase3",
-                "## gnomADg version v3.1.2",
-                "## COSMIC version 97",
-                "## gencode version GENCODE 44",
-                "## sift version 6.2.1",
-                "## genebuild version 2014-07",
-                "## assembly version GRCh38.p14",
-                "## dbSNP version 154",
-                "## ClinVar version 202301",
-                "## polyphen version 2.2.3",
-                "## gnomADe version r2.1.1",
-                "## regbuild version 1.0",
-                "## HGMD-PUBLIC version 20204",
                 "## Column descriptions:",
```

Comment:
why is this removed?

Reply:
This part of the VEP output is unfortunately not stable, so every time I rerun the tests these lines will be in a different order.